Python/Module3_IntroducingNumpy/AdvancedIndexing.md

```diff
@@ -695,6 +695,16 @@ This will return a tuple of two integer-valued index-arrays. These contain the i
 ...               [ 0.84,  0.76,  0.25,  0.07]])
 
 >>> x[np.arange(4), np.arange(4)] = range(4)
+# equivalent (works for the general case of a square matrix of N-dims)
+# x[tuple(map(np.arange, x.shape))] = range(x.shape[0])
+
+# same as above, using a tuple comprehension
```

Comment:
```suggestion
```

Reply:
I'd prefer to exclude the use of `map`, since it is not discussed elsewhere in PLYMI. The tuple comprehension is a nice substitute

Reply:
Makes sense to me!